metadata.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Daniela Plascencia
```

Comment:
I think the copyright notice should be for Canonical? Maybe @evilnick knows

Reply:
Those are inherited from 'charmcraft init', will edit/remove them.